modules/__tests__/filters.test.js

```diff
@@ -124,3 +125,10 @@ describe('isArray', () => {
         expect(isArray({ prop: 'thing' })).toBeFalsy();
     });
 });
+
+describe('widont', () => {
```

Comment:
🤣

Reply:
This pun is cannon 😅 https://www.google.co.uk/search?q=widont&oq=widont&aqs=chrome..69i57j0j69i60l3j0.648j0j1&sourceid=chrome&ie=UTF-8